enterprise/frontend/src/embedding-sdk/README.md

```diff
@@ -187,7 +195,9 @@ export default function App() {
 
   return (
     <MetabaseProvider config={config}>
-      <StaticQuestion questionId={questionId} showVisualizationSelector={false} />
+      <div style={{ width: 800, height: 600, display: "flex" }}>
```

Comment:
I'm not sure if there should be more explanation here. e.g. our chart has 0px height by default so the parent container has to have some height.

Reply:
I think a short explanation is necessary, even if temporary. Could be included in a comment inside of the code.

Reply:
1bed7c466335bd7456d6a3c0bbeb589cc28f1029